plugins/secret-squirrel/readme.md

```diff
@@ -0,0 +1,37 @@
+# @dotcom-tool-kit/secret-squirrel
+
+Tool Kit plugin to run [Secret Squirrel](https://github.com/financial-times/secret-squirrel)
+
+## Installation & Usage
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/secret-squirrel
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yml
+plugins:
+  - '@dotcom-tool-kit/secret-squirrel'
+```
+
+You will need a plugin that provides a hook to run the `SecretSquirrel` task. By default, the task runs on the `git:precommit` hook. One plugin that provides that hook is [`husky-npm`](../husky-npm); installing that alongside this plugin will enforce Secret Squirrel running on before every commit.
```

Comment:
**praise**: good to see docs being added at the same time a plugin is made, and with genuinely useful advice too!